charts/create-secret-manager-secret/templates/vaultsecret.yaml

```diff
@@ -19,3 +19,27 @@ spec:
       key: {{ .vaultKey }}
     {{- end }}
 {{- end -}}
+
+{{- range .Values.externalSecrets.secrets }}
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: {{ .secretName }}
+  labels: {{- include "datarepo.labels" . | nindent 4 }}
```

Comment:
I don't think there are labels here -- there might be? Not sure, the old secretDefintion didn't have them